addons/website_event_exhibitor/models/event_event.py

```diff
@@ -25,6 +26,13 @@ def _compute_sponsor_count(self):
         for event in self:
             event.sponsor_count = result.get(event.id, 0)
 
+    def _compute_exhibitor_count(self):
+        data = self.env['event.sponsor'].read_group([('exhibitor_type', '=', 'exhibitor')],
```

Comment:
Domain should be [('event_id', 'in', self.ids), ('exhibitor_type', '=', 'exhibitor')]. Indeed, no need to search all sponsors then filter only on current event. Note that if self.ids is void you may skip the read_group directly, to save some queries.

Reply:
Domain and method changed to not perform not needed queries if self.ids is void.